lib/matplotlib/sphinxext/plot_directive.py

```diff
@@ -254,6 +254,13 @@ def run(self):
             raise self.error(str(e))
 
 
+def _copy_css_file(app, exc):
```

Comment:
As found in https://github.com/sphinx-doc/sphinx/blob/3.x/sphinx/ext/graphviz.py#L390